packages/data-fetcher/README.md

```diff
@@ -0,0 +1 @@
+# zkSync Era Block Explorer Data Fetcher
```

Comment:
TODO: add readme

Reply:
Agreed, I want to test everything first and then add docs in the following PR.